lib/cryptoexchange/exchanges/bitexbook/services/market.rb

```diff
@@ -0,0 +1,45 @@
+module Cryptoexchange::Exchanges
+  module Bitexbook
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Bitexbook::Market::API_URL}/symbols/statistic?last_day=true"
+        end
+
+        def adapt_all(output)
+          timestamp = HashHelper.dig(output, 'meta', 'server_time')
```

Comment:
`server_time` changes but not the price. In this case, leave it blank.

Reply:
if we use minutely response then maybe we can use that timestamp